src/main/groovy/org/labkey/gradle/plugin/extension/UiTestExtension.groovy

```diff
@@ -80,8 +82,17 @@ class UiTestExtension
         }
 
         if (project.findProject(BuildUtils.getTestProjectPath(project.gradle)) != null)
-        // read test.properties file
-            PropertiesUtils.readProperties(project.project(BuildUtils.getTestProjectPath(project.gradle)).file(propertiesFile), this.config)
+        {
+            def propertiesFile = project.project(BuildUtils.getTestProjectPath(project.gradle)).file(propertiesFileName)
+            if (!propertiesFile.exists())
+            {
+                // Create test.properties file if necessary
+                def propertiesDistFile = project.project(BuildUtils.getTestProjectPath(project.gradle)).file(propertiesDistFileName)
```

Comment:
What is this file doesn't exist?

Reply:
Added a check. It will now just pass through if the file is missing.